.github/workflows/package_for_test.yml

```diff
@@ -110,6 +110,14 @@ jobs:
         env:
           CI: false
 
+      - name: Write electron-build Test yml
+        uses: actions/github-script@v6
+        with:
+          script: |
+            const fs = require('node:fs')
+            const ympPath = 'packages/neuron-wallet/electron-builder.yml'
+            fs.writeFileSync(ympPath, fs.readFileSync(ympPath).toString().replace('asar: true', 'asar: false'))
```

Comment:
I'd suggest adding a comment in `electron-builder.yml` to make the maintainers notice the source code can be included in a development build

https://github.com/nervosnetwork/neuron/blob/6640e21ea970da3913ccf826ab83d3aea6f646db/packages/neuron-wallet/electron-builder.yml#L5C7-L5C11

Reply:
Added at https://github.com/nervosnetwork/neuron/pull/2933/files#diff-510827a6f662e6ba2ea07421fbed486a0a38d996a85a9bd3c73a0e241d6f62a7R5